stl/inc/stacktrace

```diff
@@ -281,8 +281,8 @@ private:
 
     // <NOTICE> ?_Try_frames may be subject to further decreasements?
 
-    static constexpr size_t _Try_frames = 128; // Sufficient in most cases.
-    static constexpr size_t _Max_frames = 0xFFFF;
+    static constexpr size_type _Try_frames = 128; // Sufficient in most cases.
+    static constexpr size_type _Max_frames = 0xFFFF;

```

Comment:
Is this a positive change? I'm generally confused by when to use `size_t` vs `size_type`